src/Codeception/Lib/ModuleContainer.php

```diff
@@ -185,18 +177,38 @@ public function injectDependentModule($name, DependsOnModule $module)
     public function validateConflicts()
     {
         $moduleNames = array_keys($this->modules);
-        for ($i = 0; $i < count($this->modules); $i++) {
-            /** @var $currentModule Module  **/
-            $currentModule = $this->modules[$moduleNames[$i]];
+        foreach ($moduleNames as $moduleName) {
+            $currentModule = $this->modules[$moduleName];
+            /** @var $currentModule Module  * */
             if (!$currentModule instanceof ConflictsWithModule) {
-                continue;
+                continue; // don't validate modules which are not in conflict
+            }
+            if ($currentModule instanceof PartedModule) {
```

Comment:
Well, that's code style, of course but to me `&&` reads better than nested `if`s.